apps/web/app/(app)/environments/[environmentId]/settings/(organization)/members/components/EditMemberships/OrganizationActions.tsx

```diff
@@ -83,14 +87,23 @@ export const OrganizationActions = ({
             Leave organization
           </Button>
         )}
-        <Button
-          variant="secondary"
-          size="sm"
-          onClick={() => {
-            setCreateOrganizationModalOpen(true);
-          }}>
-          Create new organization
-        </Button>
+        <TooltipRenderer
+          shouldRender={!isMultiOrgEnabled}
+          tooltipContent="Upgrade your enterprise license to create multiple organizations">
```

Comment:
Please remove the button completely if `!isMultiOrgEnabled` since it's not possible for users to upgrade their license to create multiple organizations. This is a Formbricks Cloud only feature.